apps/nextcloud/templates/services/nginx/dc.yml.j2

```diff
@@ -69,7 +69,7 @@ spec:
             periodSeconds: 5
 
       initContainers:
-        - image: "{{ internal_docker_registry }}/{{ project_name }}/nextcloud:{{ nextcloud_image_tag }}"
```

Comment:
missing imagePullSecrets?

Reply:
`imagePullSecrets` is at the same level, you define one for your deploymentconfig, is it possible to have several ?

Reply:
The imagePullSecrets is a list so I suppose you can have several and they either match it by tld or by trying them successively? My point is that, in this case, the images used by the container and by the initContainer are different and come from different registries :man_shrugging:

Reply:
Can you give me the documentation ? In this schema I don't understand this is a list https://docs.openshift.com/container-platform/3.9/rest_api/oapi/v1.DeploymentConfig.html#object-schema
But I agree with you, here the image can come from different registries.

Reply:
SO has this: https://stackoverflow.com/questions/54717135/are-multiple-imagepullsecrets-allowed-and-used-by-kubernetes-to-pull-an-image-fr It seems to be a feature for High Availability of docker registries but should also allow to pull images from different private registries like you present case.

Reply:
:pray: I made the modification